### PR TITLE
(GH-3) Add StyleCop analyzer to test assemebly

### LINKED ISSUE #3: Add analyzer to test assembly
Define ruleset for test assembly and add analyzer to the project

### PR DESCRIPTION
**Pull request by pascalberger**
Adds StyleCop analyzer to test assemebly and defines separate ruleset for test assemblies with less strict validation.

Fixes #3

### CODE REVIEW

File: src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Cake.Prca.Issues.MsBuild.XML</DocumentationFile>
-    <CodeAnalysisRuleSet>..\Cake.Prca.Issues.MsBuild.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>..\Cake.Prca.Issues.MsBuild.Tests.ruleset</CodeAnalysisRuleSet>
```
Comment (stffabi) [inline src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj]: Here we don't want to use the Tests ruleset.

File: src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj [hunk 2]
```diff
@@ -33,7 +33,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\Cake.Prca.Issues.MsBuild.XML</DocumentationFile>
-    <CodeAnalysisRuleSet>..\Cake.Prca.Issues.MsBuild.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>..\Cake.Prca.Issues.MsBuild.Tests.ruleset</CodeAnalysisRuleSet>
```
Comment (stffabi) [inline src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj]: Here we don't want to use the Tests ruleset.

File: src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj (hunk 1 above)
Comment (pascalberger) [inline src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj]: Fixed

File: src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj (hunk 2 above)
Comment (pascalberger) [inline src/Cake.Prca.Issues.MsBuild/Cake.Prca.Issues.MsBuild.csproj]: Fixed

**Pull request merged**